Tests/DoNilDisturbPluginTests/DoNotDisturbPluginTests.swift

```diff
@@ -0,0 +1,65 @@
+import XCTest
+import Foundation
+import iCalendar
+
+@testable import PluginBinary
+
+final class DoNotDisturbPluginTests: XCTestCase {
+    
+    func testHolidayLoad() throws {
+        let date = Foundation.Calendar.current.date(from: DateComponents(year: 2022, month: 8, day: 30, hour: 13))!
+        let path = #filePath
```

Comment:
Manually constructing paths isn't safe in tests. Could you please use the API for loading bundle resources? https://developer.apple.com/documentation/xcode/bundling-resources-with-a-swift-package

Reply:
yep can do

Reply:
Much appreciated